src/main/java/org/jullaene/walkmong_back/api/dog/rest/DogController.java

```diff
@@ -10,4 +15,10 @@
 @RequestMapping("/api/v1/dog")
 public class DogController {
     private final DogService dogService;
+
+    @GetMapping("/profile/{dogId}")
+    public ResponseMessage<DogProfileResponseDto> getDogProfile(@PathVariable Long dogId) {
+        DogProfileResponseDto dogProfile = dogService.getDogProfile(dogId);
+        return ResponseMessage.of(HttpStatus.OK, "Success", dogProfile);
```

Comment:
```suggestion
    public ResponseMessage<BasicResponse<DogProfileResponseDto>> getDogProfile(@PathVariable Long dogId) {
        DogProfileResponseDto dogProfile = dogService.getDogProfile(dogId);
        return ResponseMessage.of(BasicResponse.ofSuccess(dogProfile));
```

이렇게 해주시면 저희 기본 반환 형태로 반환됩니다

Reply:
이 부분 ResponseMessage 말고 ResponseEntity 사용하는걸로 바꿨는데 확인 부탁드립니다!